packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -70,9 +426,15 @@ class WidgetInspector extends StatefulWidget {
 class _WidgetInspectorState extends State<WidgetInspector>
     with WidgetsBindingObserver {
 
+  _WidgetInspectorState() : selection = WidgetInspectorService.instance.selection {
+    WidgetInspectorService.instance.selectionChangedCallback = () {
+      setState(() {});
```

Comment:
idiomatically, this probably belongs in initState rather than a constructor

Reply:
sounds good. done.